spec/features/admin_deletes_pizzeria_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "admin can delete pizzerias", %{
+  As a admin
+  I want to delete pizzerias
+  So that I can keep content updated
+
+  Acceptance Criteria
+  * Admin must be able to delete pizzerias from admin panel
+  } do
+
+  scenario "admin can delete a pizzeria" do
+    pizzerias = FactoryGirl.create_list(:pizzeria, 10)
+
+    admin = FactoryGirl.create(:user, admin: true)
+
+    sign_in(admin)
+
+    visit admin_pizzerias_path
+
+    first(".pizzeria").click_link("Delete pizzeria")
+
+    expect(page).to have_content "Pizzeria Deleted"
+    expect(pizzerias.count).to eq 10
```

Comment:
Same question here as with comments, if we create 10 and delete one, why are expecting to still have 10?